### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ArturoTorresMartinez**
This is the youtube pretend template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="/assets/stylesheets/main.css">
+  </hea>
```
Comment (JamesOkunlade) [inline index.html]: I believe you have no use for this line. Please remove it.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+<body>
+  <header class="main-header">
+    <nav class="main-header__nav">
+      <ul class="main-header__item-list">
+        <li class="main-header__items"> <img src="/assets/images/playButton.png" height="30px"><a
```
Comment (JamesOkunlade) [inline index.html]: You really should not have to use inline styling for the img here unless it's unavoidable. Give similar images a common class name and style it height in your CSS file. Also your img element should always contain the alt="" attribute.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            class="active">Youtube</a><input type="text" placeholder="Search" class="searchBox">
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="flex-container">
+    <div class="player">
+      <div class="player-video">
+        <iframe width="896px" height="504px" src="https://www.youtube.com/embed/vioOIXrOAa0" frameborder="0"
```
Comment (JamesOkunlade) [inline index.html]: You should remove all the inline styles you have in your code and do all the styling in your CSS file. Remove the width, height, and frameborder.

File: index.html (hunk 1 above)
Comment (ArturoTorresMartinez) [reply index.html]: What line, you highlighted 4 of them

Comment (ArturoTorresMartinez) [reply index.html]: > I believe you have no use for this line. Please remove it.

What line? you highlighted four of them

**Pull request closed without merging**